sites/partners/src/components/listings/PaperListingForm/sections/ApplicationTypes.tsx

```diff
@@ -289,39 +302,37 @@ const ApplicationTypes = ({ listing }: { listing: FormListing }) => {
             </Grid.Cell>
           )}
         </Grid.Row>
-        {((commonDigitalApplicationChoice &&
-          commonDigitalApplicationChoice === YesNoEnum.no &&
-          digitalApplicationChoice === YesNoEnum.yes) ||
-          (digitalApplicationChoice === YesNoEnum.yes &&
-            !commonDigitalApplicationChoice &&
-            listing?.commonDigitalApplication === false)) && (
-          <Grid.Row columns={1}>
-            <Grid.Cell>
-              <Field
-                label={t("listings.customOnlineApplicationUrl")}
-                name="customOnlineApplicationUrl"
-                id="customOnlineApplicationUrl"
-                placeholder="https://"
-                inputProps={{
-                  value: methods.digital?.externalReference
-                    ? methods.digital.externalReference
-                    : "",
-                  onChange: (e) => {
-                    setMethods({
-                      ...methods,
-                      digital: {
-                        ...methods.digital,
-                        externalReference: e.target.value,
-                      },
-                    })
-                  },
-                }}
-                error={fieldHasError(errors?.applicationMethods?.[0]?.externalReference)}
-                errorMessage={fieldMessage(errors?.applicationMethods?.[0]?.externalReference)}
-              />
-            </Grid.Cell>
-          </Grid.Row>
-        )}
+        {digitalApplicationChoice === YesNoEnum.yes &&
+          (!disableCommonApplication ||
```

Comment:
Issue: this field should appear in the two scenarios:
* `Is there a digital application?` is set to "Yes" but `Are you using the common digital application?` is set to "No"
* `disableCommonApplication` feature flag is turned on for the jurisdiction and `Is there a digital Application?` is set to "Yes"


Yes

No
Required to publish